tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -42,6 +44,85 @@ type stresser struct {
 	KeySuffixRange int
 
 	N int
+
+	mu      sync.Mutex
+	failure int
+	success int
+
+	cancel func()
+}
+
+func (s *stresser) Stress() error {
+	// TODO: make this configurable to take advantage of HTTP2 streams
```

Comment:
we probably do not want to do this. we can add more stresser as we want. each stresser is just a simple unit.